contracts/presets/ERC20PresetFixedSupply.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.2;
+
+import "../token/ERC20/ERC20.sol";
+import "../token/ERC20/ERC20Burnable.sol";
+
+/**
+ * @dev Extension of {ERC20} and {ERC20Burnable} with fixed initialSupply.
+ *
+ * Tip: Contract is preconfigured with fixed initial supply.
+ * minting / pausing functionality is not supported. This removes the need of access control and thereby governance. 
+ */
+contract ERC20PresetFixedSupply is ERC20, ERC20Burnable {
```

Comment:
What was the reason for including `ERC20Burnable`?

Reply:
Burn functionality would enable the use of this preset for the purposes of deflationary tokens, correct?

Reply:
Emmm, I am confused the can I just write `contract ERC20PresetFixedSupply is ERC20Burnable`, cause `ERC20Burnable` is derived from `ERC20`

Reply:
ahh, yes you are right! Thanks @Skyge